packages/@expo/prebuild-config/src/importExpoModulesAutolinking.ts

```diff
@@ -24,15 +24,14 @@ function tryRequireExpoModulesAutolinking(projectRoot: string): AutolinkingModul
   const resolveOptions = { paths: [projectRoot] };
 
   try {
-    resolvedAutolinkingPath = require.resolve('expo-modules-autolinking/exports', resolveOptions);
-  } catch {}
-  // Fallback to the older version of expo-modules-autolinking
-  try {
-    resolvedAutolinkingPath = require.resolve(
-      'expo-modules-autolinking/build/autolinking',
-      resolveOptions
-    );
-  } catch {}
+    // Autolinking is exported from the `expo` package as of SDK 52
+    resolvedAutolinkingPath = require.resolve('expo/autolinking', resolveOptions);
+  } catch {
+    // Fallback to require from `expo-modules-autolinking` on SDK 50 and 51
```

Comment:
imo this isn't really needed since you shouldn't be using sdk 52 prebuild-config with older sdk versions, but it's thoughtful and doesn't hurt